LICENSE

```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 Dylan Turner
+Copyright (c) 2019 Thomas Grainger tagrain@gmail.com (aka Dylan Turner)
+Copyright (c) 2025 Thomas Grainger, Hugo van Kemenade, Peter Cock and contributors
```

Comment:
We use the first year in PSF projects: https://devguide.python.org/getting-started/pull-request-lifecycle/#copyrights

And we first contributed in 2019: https://github.com/flake8-implicit-str-concat/flake8-implicit-str-concat/graphs/contributors?from=01%2F11%2F2019&to=02%2F03%2F2020

```suggestion
Copyright (c) 2019 Thomas Grainger, Hugo van Kemenade, Peter Cock and contributors
```